engines/stark/stark.cpp

```diff
@@ -70,11 +71,9 @@ StarkEngine::StarkEngine(OSystem *syst, const ADGameDescription *gameDesc) :
 		_diary(nullptr),
 		_userInterface(nullptr),
 		_fontProvider(nullptr),
+		_settings(nullptr),
 		_lastClickTime(0) {
 	_mixer->setVolumeForSoundType(Audio::Mixer::kPlainSoundType, 127);
```

Comment:
We don't need this line either as we don't use `kPlainSoundType` and `syncSoundSettings` sets it anyways.

https://github.com/residualvm/residualvm/blob/3ec01653c3e569a427f65a3e83914011ad939ae0/engines/engine.cpp#L585